params.yml

```diff
@@ -41,7 +41,7 @@ zkevm_contracts_repo: https://github.com/0xPolygonHermez/zkevm-contracts.git
 
 zkevm_agglayer_image: nulyjkdhthz/agglayer:v0.1.0
 zkevm_bridge_service_image: hermeznetwork/zkevm-bridge-service:v0.4.2
-zkevm_bridge_ui_image: hermeznetwork/zkevm-bridge-ui:latest # TODO: better tags for the bridge ui
+zkevm_bridge_ui_image: hermeznetwork/zkevm-bridge-ui:multi-network
```

Comment:
nice!

Reply:
By the way, the bridge-service image is hardcoded, it would be great to put it here too.
https://github.com/0xPolygon/kurtosis-cdk/blob/main/cdk_bridge_infra.star#L63